aikau/src/test/resources/alfresco/core/PageTest.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2005-2016 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This test assesses the CoreRwd mixin as applied to AlfMenuBarPopup
+ *
+ * @author Richard Smith
+ * @author Dave Draper
+ */
```

Comment:
Fair point .... hadn't noticed that. Perhaps we should try and JSDoc all our files, even the test ones, as a means of documenting them for the next developer?